vue/dynamicforms/src/components/form/inputs/df-file.vue

```diff
@@ -87,11 +87,11 @@ function removeFile() {
 
 async function upload() {
   progress = 0;
-  const fileRef = file.value!;
+  const fileRef = df.value!;
```

Comment:
This does not look right and should not be right. File is a reference, while df does not exists.

Revert.